src/components/perps/Module/ConfirmationSummary.tsx

```diff
@@ -240,26 +241,31 @@ export default function ConfirmationSummary(props: Props) {
             )
             return (
               <SummaryRow label={label} key={index} className={classNames(isPnl && 'font-bold')}>
-                <DisplayCurrency
-                  className={classNames(
-                    'text-xs',
-                    isPnl && 'font-bold',
-                    isPnl && pnlAmount.isPositive() && 'text-profit',
-                    isPnl && pnlAmount.isNegative() && 'text-loss',
-                  )}
-                  coin={BNCoin.fromDenomAndBigNumber(
-                    baseDenom,
-                    isPnl && !limitPrice
-                      ? pnlAmount.minus(
-                          tradingFeeAndPrice.fee.opening.plus(tradingFeeAndPrice.fee.closing),
-                        )
-                      : pnlAmount,
-                  )}
-                  options={{
-                    abbreviated: false,
-                  }}
-                  showSignPrefix={true}
-                />
+                {isPnl ? (
+                  <PnLDisplay
+                    pnlAmount={pnlAmount}
+                    pnlPercentage={(() => {
+                      if (pnlAmount.isZero() || !position.size || !position.entry_price)
+                        return BN_ZERO
+
+                      const positionValue = BN(position.size).abs().times(position.entry_price)
```

Comment:
Better to name to entryValue. PositionValue would be with the current price

```suggestion
                      const entryValue = BN(position.size).abs().times(position.entry_price)
```